src/main/java/com/revature/ams/Booking/dtos/BookingResponseDTO.java

```diff
@@ -8,6 +8,11 @@
 
 /**
  * TODO: DOCUMENT ME
+ * Making a class called BookingsResponseDTO which declares data as well as implements 3 constructors.
```

Comment:
I only see one constructor in this file, can you specify?

Reply:
We are overloading constructors 
First constructor is 
public BookingResponseDTO() {
 }
 
 Second constructor
   public BookingResponseDTO(int bookingId, Flight flight, boolean carryOnAllowed, short checkedLuggage, String seatType, BigDecimal price) {
        this.bookingId = bookingId;
        this.flight = flight;
        this.carryOnAllowed = carryOnAllowed;
        this.checkedLuggage = checkedLuggage;
        this.seatType = seatType;
        this.price = price;
    }
    
  Third constructor which is being used throughout application
   public BookingResponseDTO(Booking booking){
        this.flight = booking.getFlight();
        this.bookingId = booking.getBookingId();
        this.carryOnAllowed = booking.isCarryOnAllowed();
        this.seatType = booking.getSeatType().name();
        this.price = booking.getPrice();
        this.checkedLuggage = booking.getCheckedLuggage();
    }